tests/namespacing/success/vector.toml

```diff
@@ -1,6 +1,11 @@
 # Set global options
 data_dir = "/var/lib/vector"
 
```

Comment:
can we add "processing" pipelines here as an input to any sink/transform? 
If I understand correctly now it's not connected to any output.